content/en/docs/setup/upgrade/notice/index.md

```diff
@@ -1,44 +1,70 @@
 compatibility.  We also mention cases where backwards compatibility was
 preserved but new behavior was introduced that would be surprising to someone
-familiar with the use and operation of Istio 1.1.
+familiar with the use and operation of Istio 1.2.
 
 For an overview of new features introduced with Istio 1.2, please refer
-to the [1.2 release notes](/about/notes/1.2/).
-
-## Installation and Upgrade
-
-{{< tip >}}
-The configuration model for Mixer has been simplified. Support for
-adapter-specific and template-specific Custom Resources has been
-removed by default in 1.2 and will be removed entirely in 1.3.
-Please move to the new configuration model.
-{{< /tip >}}
-
-Most Mixer CRDs were removed from the system to simplify the configuration
-model, improve performance of Mixer when used with Kubernetes, and improve
-reliability in a variety of Kubernetes environments.
-
-The following CRDs remain:
-
-| Custom Resource Definition name | Purpose |
-| --- | --- |
-| `adapter`| Specification of Istio extension declarations |
-| `attributemanifest` | Specification of Istio extension declarations |
-| `template` | Specification of Istio extension declarations |
-| `handler` | Specification of extension invocations |
-| `rule` | Specification of extension invocations |
-| `instance` | Specification of extension invocations |
-
-In the event you are using the removed mixer configuration schemas, set
-the following Helm flags during upgrade of the main Helm chart:
-`--set mixer.templates.useTemplateCRDs=true --set mixer.adapters.useAdapterCRDs=true`
+to the [1.3 release notes](/about/notes/1.3/).
+
+## Installation and upgrade
+
+We simplified the configuration model for Mixer and removed support for
+adapter-specific and template-specific Custom Resource Definitions (CRDs)
+entirely in 1.3. Please move to the new configuration model.
+
+We removed the Mixer CRDs from the system to simplify the configuration
+model, improve Mixer's performance in Kubernetes deployments, and improve
+reliability in various Kubernetes environments.
+
+## Traffic management
+
+Istio now captures all ports by default. If you don't specify container ports
+to intentionally bypass Envoy, you must opt out of port capturing with the
+`traffic.sidecar.istio.io/excludeInboundPorts` option.
+
+{{< warning >}}
+This change exposes any applications listening on `127.0.0.1` since Envoy
+connects over localhost. Opt out to avoid exposing such applications.
+{{< /warning >}}
+
+Protocol sniffing is now enabled by default. Disable protocol sniffing with the
+`--set pilot.enableProtocolSniffing=false` option when you upgrade to get the
+previous behavior. To learn more see our [protocol selection page](/docs/ops/traffic-management/protocol-selection/).
+
+To specify a hostname in multiple namespaces, you must select a single host using
+a [`Sidecar` resource](/docs/reference/config/networking/v1alpha3/sidecar/).
+
+## Trust domain validation
```

Comment:
Thanks for integrating this, could also integrate the changes in `docs/tasks/security/authn-policy/index.md` from https://github.com/istio/istio.io/pull/4908 to this PR? I think we don't need a separate PR.

cc @myidpt